spec/lib/dradis/plugins/projects/export/v2/template_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+describe Dradis::Plugins::Projects::Export::V2::Template do
+  let(:project) { create(:project) }
+  let(:user) { create(:user) }
+  let(:export) do
+    described_class.new(
+      default_user_id: user.id,
+      plugin: Dradis::Plugins::Projects,
+      project_id: project.id
+    ).export
+  end
+
+  context 'exporting a project' do
+    context 'with comments in an issue' do
```

Comment:
We can probably combine these two `context` and save some time as they aren't mutually exclusive, instead of exporting twice.

Reply:
let's not spent too much time fixing these specs, we may need to migrate this to use the `combustion` gem